### DISCUSSION TITLE
Integrating with external ReactJS

### DISCUSSION THREAD

**Discussion by MrVibe** [xyflow/xyflow#2888] [General]
So I have an existing React 17 project and I want to use this module without including this in the core project. 
When I include it shows multiple react versions. 
When I modify my Webpack for the reactflow to use external react it shows react runtime security error. [__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED ] 

 Can anyone guide me how to use this with external react ?

**Comment by moklick** [comment]
Which version of React Flow are you trying to install?
